docs_user/assemblies/assembly_network-requirements-and-configuration.adoc

```diff
@@ -0,0 +1,51 @@
+[id="network-requirements-and-configuration_{context}"]
+
+:context: network-requirements
+
+= Network requirements and configuration
```

Comment:
The title of this assembly looks like a conceptual module (and maybe one should be included with much of this leading information). But includes other assemblies with gerunds and a process module. 
Is this the best title here?

Reply:
Maybe something like, "Configuring the network for the RHOSO deployment"?

Reply:
I was never good in finding names or titles but it "Configuring the network for the RHOSO deployment" sounds good for me